cmd/app/settings.go

```diff
@@ -0,0 +1,121 @@
+	"github.com/slackapi/slack-cli/internal/slacktrace"
+	"github.com/slackapi/slack-cli/internal/style"
+	"github.com/spf13/cobra"
+)
+
+var settingsAppSelectPromptFunc = prompts.AppSelectPrompt
+
+func NewSettingsCommand(clients *shared.ClientFactory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "settings [flags]",
+		Short: "Open app settings for configurations",
+		Long: strings.Join([]string{
+			"Open app settings to configure an application in a web browser.",
+			"",
+			"Discovering new features and customizing an app manifest can be done from this",
+			fmt.Sprintf("intuitive interface for apps with a \"%s\" manifest source.", config.ManifestSourceRemote.String()),
+			"",
+			"This command does not support apps deployed to Run on Slack infrastructure.",
+		}, "\n"),
+		Example: style.ExampleCommandsf([]style.ExampleCommand{
+			{
+				Meaning: "Open app settings for a prompted app",
+				Command: "app settings",
+			},
+			{
+				Meaning: "Open app settings for a specific app",
+				Command: "app settings --app A0123456789",
+			},
+		}),
+		Args: cobra.MaximumNArgs(0),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return appSettingsCommandPreRunE(clients, cmd, args)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return appSettingsCommandRunE(clients, cmd, args)
+		},
+	}
+	return cmd
+}
+
+// appSettingsCommandPreRunE determines if the command can be run in a project
+func appSettingsCommandPreRunE(clients *shared.ClientFactory, cmd *cobra.Command, args []string) error {
+	ctx := cmd.Context()
+	err := cmdutil.IsValidProjectDirectory(clients)
+	if err != nil {
+		return err
+	}
+	if clients.Config.ForceFlag {
+		return nil
+	}
+	err = cmdutil.IsSlackHostedProject(ctx, clients)
+	if err != nil {
+		if slackerror.Is(err, slackerror.ErrAppNotHosted) {
+			return nil
+		} else {
+			return err
+		}
+	}
+	return slackerror.New(slackerror.ErrAppHosted).
+		WithDetails(slackerror.ErrorDetails{
+			{
+				Message: "App settings is not supported with Run on Slack infrastructure",
+			},
+		})
+}
+
+// appSettingsCommandRunE opens app settings in a browser for the selected app
+func appSettingsCommandRunE(clients *shared.ClientFactory, cmd *cobra.Command, args []string) error {
+	ctx := cmd.Context()
+	clients.IO.PrintTrace(ctx, slacktrace.AppSettingsStart)
+
+	app, err := settingsAppSelectPromptFunc(ctx, clients, prompts.ShowInstalledAndUninstalledApps)
+	if err != nil {
+		return err
+	}
+	host := clients.API().Host()
+	parsed, err := url.Parse(host)
+	if err != nil {
+		return err
+	}
+	parsed.Host = "api." + parsed.Host
+	settingsURL := fmt.Sprintf("%s/apps/%s", parsed.String(), app.App.AppID)
+
+	clients.IO.PrintInfo(ctx, false, "\n%s", style.Sectionf(style.TextSection{
+		Emoji: "house",
+		Text:  "App Settings",
+		Secondary: []string{
+			settingsURL,
+		},
+	}))
+	clients.Browser().OpenURL(settingsURL)
+
+	clients.IO.PrintTrace(ctx, slacktrace.AppSettingsSuccess, settingsURL)
```

Comment:
praise: heh, you read my mind to add `settingsURL` 😉

Reply:
The setups of `PrintTrace` continue to impress me! 👾

I'm hoping soon we can equip all commands with at least a simple "start" and "success" with follow ups in `@slack/cli-test` but for now I'm also so glad we can print a success message with additional detail 📚 ✨

Reply:
It seems like a reasonable PR to add a "start" and "success" to each command and land the equivalent definitions in `@slack/cli-test`.